sda/config_local.yaml

```
@@ -0,0 +1,108 @@
+log:
+  format: "json"
+  level: "debug"
+api:
+  rbacFile: ../.github/integration/sda/rbac.json 
+
+archive:
+  type: s3
+  url: "http://localhost"
+  port: 19000
+  readypath: "/minio/health/ready"
+  accessKey: "access"
+  secretKey: "secretKey"
+  bucket: "archive"
+  region: "us-east-1"
+
+auth:
+  cega:
+    authUrl: "http://localhost:8443/username/"
+    id:
+    secret:
+  infoText: "About service text"
+  infoURL: "http://example.org/about"
+  jwt:
+    issuer: "https://localhost:8888"
+    privateKey: "tmp/shared/keys/jwt.key"
```

```suggestion
    privateKey: "/tmp/shared/keys/jwt.key"
```
And similar for the other cases.
Needed for me to make the services run.

---

Good catch! fixed in deaf0a34